lib/node/nodes/aggregate-intersection.js

```diff
@@ -77,3 +77,16 @@ var queryAggregateTemplate = Node.template([
     'WHERE ST_Intersects(_cdb_analysis_source.the_geom, _cdb_analysis_target.the_geom)',
     'GROUP BY {{=it.groupByColumns}}'
 ].join('\n'));
+
+AggregateIntersection.prototype.computeRequirements = function(databaseService, limits, callback) {
+    // we estimate the maximum possible number of rows of the result
+    var product = this.source.estimatedRequirements.numberOfRows *
```

Comment:
I guess this is not consider good practice, should I define an accessor and use `this.source.getEstimatedRequirements().numberOfRows` instead?
(your opinion will be welcome, @dgaubert )